codecs/avif/dec/avif_dec.cpp

```diff
@@ -0,0 +1,59 @@
+#include <emscripten/bind.h>
+#include <emscripten/val.h>
+#include "avif/avif.h"
+
+using namespace emscripten;
+
+class RawImage {
```

Comment:
@surma Please remove this in favour of `ImageData` using other codecs as examples.